earthfile2llb/with_docker_run_reg.go

```diff
@@ -320,3 +321,18 @@ func (w *withDockerRunRegistry) load(ctx context.Context, opt DockerLoadOpt) (ch
 
 	return imageDefChan, nil
 }
+
+func readImgResults(ctx context.Context, ch chan *states.ImageResult) ([]*states.ImageResult, error) {
+	var results []*states.ImageResult
+	for {
+		select {
+		case result, ok := <-ch:
+			if !ok {
```

Comment:
observation: the expected (non-error) code flow requires this to be "not ok" :laughing:

Reply:
(this isn't even a nit, just an observation I thought was amusing)

Reply:
Hah, that is amusing. It's tempting to use `keepGoing` or something, but I think convention makes it reasonable to still use `ok` in spite of the ... odd flow.